src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -190,23 +190,32 @@ public async Task UseCallback_NotSecureConnection_CallbackNotCalled()
 
         public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesDuringCallback_Urls()
         {
-            foreach (bool checkRevocation in new[] { true, false })
+            foreach (Configuration.Http.RemoteServer remoteServer in Configuration.Http.RemoteServers)
             {
-                yield return new object[] { Configuration.Http.SecureRemoteEchoServer, checkRevocation };
-                yield return new object[] {
-                    Configuration.Http.RedirectUriForDestinationUri(
-                        secure:true,
-                        statusCode:302,
-                        destinationUri:Configuration.Http.SecureRemoteEchoServer,
-                        hops:1),
-                    checkRevocation };
+                if (remoteServer.IsSecure)
+                {
+                    foreach (bool checkRevocation in new[] { true, false })
+                    {
+                        yield return new object[] {
+                            remoteServer,
+                            remoteServer.EchoUri,
+                            checkRevocation };
+                        yield return new object[] {
+                            remoteServer,
+                            remoteServer.RedirectUriForDestinationUri(
+                                statusCode:302,
```

Comment:
nit: add space after ":" for named parameters.

(comment applies to other places in the PR)